widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/CustomCardViewHolder.java

```diff
@@ -0,0 +1,41 @@
+package com.glia.widgets.chat.adapter.holder;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.glia.androidsdk.chat.ChatMessage;
+
+/**
+ * A CustomMessageViewHolder describes an item view
+ * <p>
+ * CustomMessageViewHolder implementation should have fields
```

Comment:
Maybe `CustomMessageViewHolder` should  be renamed to `CustomCardViewHolder` (class name)?

Reply:
fixed